packages/apt/test.sh

```diff
@@ -165,11 +165,20 @@ esac
 
 # Upgrade
 if [ -n "${old_package}" ]; then
+  cd
   sudo apt purge -V -y \
     ${package} \
     "${mysql_package_prefix}-*"
   sudo rm -rf /var/lib/mysql
   sudo mv /etc/apt/sources.list.d/${package}.list /tmp/
+  case ${package} in
+    mysql-community-8.*)
+      sudo \
+        env DEBIAN_FRONTEND=noninteractive \
+            MYSQL_SERVER_VERSION=mysql-${mysql_version} \
+          apt install -y ./mysql-apt-config_*_all.deb
```

Comment:
Why is `mysql-apt-config*.deb` removed? Is `"${mysql_package_prefix}-*" matched?

Reply:
`${mysql_package_prefix}` is `mysql` and it matches `mysql-apt-config`:

https://github.com/mroonga/mroonga/runs/7985783662?check_suite_focus=true#step:5:2377

```text
+ sudo apt purge -V -y mysql-community-8.0-mroonga 'mysql-*'
```

https://github.com/mroonga/mroonga/runs/7985783662?check_suite_focus=true#step:5:2476

```text
   mysql-apt-config* (0.8.22-1)
```

Reply:
Yes, `mysql-apt-config*.deb` matched "${mysql_package_prefix}-*".
Therefore, `mysql-apt-config` is removed by `sudo apt purge -V -y mysql-community-8.0-mroonga 'mysql-*'`.